doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2551,14 +2583,7 @@ Once you have a username from DataCite, you can enter it like this:
 Legacy Single PID Provider: dataverse.pid.datacite.password
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Once you have a password from your provider, you should create a password alias.
-This avoids storing it in clear text, although you could use a JVM option `to reference
-a different place <https://docs.payara.fish/community/docs/Technical%20Documentation/Payara%20Server%20Documentation/Server%20Configuration%20And%20Management/Configuration%20Options/Variable%20Substitution/Types%20of%20Variables.html>`__.
-
-``./asadmin create-password-alias dataverse.pid.datacite.password``
```

Comment:
Sorry but I don't get what you're asking? I introduced a new section in the config guide specifically about securing your secrets. So instead of repeating the same thing over and over, there is a single place where the different options are described. Why would you still like to keep an additional example around?

Reply:
Just in case: we *could* add a template that takes a name argument and repeats the options at the various places. We could even use the tabs extension as we do in the container dev guide.

Reply:
Yeah, it makes sense what you're doing, especially for poor souls like us that have to maintain the documentation. I hope the sysadmins can follow along easily enough.

Reply:
Right. Suppose time will tell. Should we resolve this for now?

Reply:
Yes, sure. Will do.